README.md

```diff
@@ -0,0 +1,41 @@
+# PC98 Image Tools
+Tools for working with the PC98 hard drive images used by common emulators.
+
+## Formats Supported
+ * HDD
+ * HDI
+ * NHD
+ * SLH
+
+See [`formats`](formats/) for the various format structures.
+
+## Tools
+### `hdddump.py`
+Display header information from an HDD hard drive image file.
+
+Verifies magic values in header.
+
+### `hddgen.py`
+Creates a new blank HDD image file to the user's specifications.
```

Comment:
All `*gen.py` scripts actually only create image *headers*, not the images themselves. I'd imagine most emulators would expect some blank space after the header.